lib/game.rb

```diff
@@ -22,14 +26,14 @@ def initialize
 
   def game_start
     computer_place_ships
-    @game_message.computer_board_placement(computer_board.board)
+    @game_message.computer_board_placement(@computer_board)
     player_place_ships
     take_turn
   end
 
   def computer_place_ships
-    @computer_board.computer_place_ship(computer_cruiser)
-    @computer_board.computer_place_ship(computer_sub)
+    @board_setup.computer_place_ship(computer_board, computer_cruiser)
+    @board_setup.computer_place_ship(computer_board, computer_sub)
```

Comment:
This makes change makes the code much clearer. You are calling on the action in board_setup class to establish your board in the GAME. Good thinking on this one!